docs/java-rest/high-level/apis/index.asciidoc

```diff
@@ -1,6 +1,12 @@
 include::create_index.asciidoc[]
 
+<<<<<<< HEAD
+include::indices_exists.asciidoc[]
+
+include::deleteindex.asciidoc[]
+=======
 include::delete_index.asciidoc[]
+>>>>>>> master
```

Comment:
could you fix this? A merge gone wrong I think

Reply:
include::deleteindex.asciidoc[] should go, it's been replaced by include::delete_index.asciidoc[]

Reply:
Fixed, thanks for noticing it!